products/sle/main.pm

```diff
@@ -601,6 +601,7 @@ sub load_consoletests() {
                 loadtest "console/installation_snapshots";
             }
             loadtest "console/snapper_undochange";
+            loadtest "console/snapper_create";
```

Comment:
Fixed & enabled in PR https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/2453.